docs/source/guide/guide-getting-started.rst

```diff
@@ -262,3 +266,63 @@ We can then use this backend for our mitigation.
 
 Note that we don't need to even redefine factories for different stacks. Once
 you have a ``Factory`` it can be used with different front and backends.
+
+Error Mitigation with Probabilistic Error Cancellation
+------------------------------------------------------
+
+In *Mitiq*, it is very easy to switch between different error mitigation methods.
+
+For example, we can implement Probabilistic Error Cancellation (PEC) by using the same execution function (``executor``)
+and the same *Cirq* circuit (``circuit``) that we have already defined in the section
+:ref:`Multi-platform Framework <multi_platform_framework>`.
+
+Differently from ZNE, PEC requires the knowledge of the noise model and of the noise strength acting on the system.
+In particular for each operation of the circuit, we need to build a quasi-probability representation of the 
+ideal unitary gate expanded in a basis of noisy implementable operations. For more details on PEC see
+the :ref:`Probabilistic Error Cancellation <guide_qem_pec>` section.
+
+In our simple case, ``circuit`` corresponds to the repetition of the same X gate,
+whose representation in the presence of depolarizing noise can be obtained as follows:
+
+.. testcode::
+
+    from mitiq.pec.representations import represent_operation_with_local_depolarizing_noise
+
+    x_representation = represent_operation_with_local_depolarizing_noise(
+        ideal_operation=Circuit(X(qubit)), 
+        noise_level=NOISE_LEVEL,
+    )
+
+    print(x_representation)
+
+.. testoutput::
+
+    0: ───X─── = 1.010*0: ───X───-0.003*0: ───X───X───-0.003*0: ───X───Y───-0.003*0: ───X───Z───
+
+The result above is an :class:`OperationRepresentation` object which contains the information for 
+representing the ideal operation X (left-hand-side of the printed output) as a linear combination of
+noisy operations (right-hand-side of the printed output). 
+
+We can now implement PEC by importing the function ``execute_with_pec`` from the 
+``mitiq.pec`` module.
```

Comment:
```suggestion
We can now implement PEC by importing the function :func:`~mitiq.pec.pec.execute_with_pec` from the
:mod:`~mitiq.pec.pec` module.
```

Reply:
Thanks Nathan I always forget about this! Now I did this for all the functions/methods/classes of the Getting Started section